### ISSUE TITLE
Unable to close on Android

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#360]
When the user is on "How COVID Alert works", they're not able to close this by using the top left arrow. 
Note that TalkBack in NOT on. 

Since this arrow is quite high on the screen, the top notifications keep popping up. The arrow is quite small and it doesn't work.

Android 8 on Samsung Galaxy

![10](https://user-images.githubusercontent.com/14280062/86377431-58212d00-bc56-11ea-93a0-631a5f6b28bf.jpg)
